lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -628,42 +629,39 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                 });
-  patterns.onOp("Not", 1,
-                [](OpBinder binder, ConversionPatternRewriter &rewriter) {
-                  Torch::ValueTensorType resultType;
-                  Value operand;
-                  if (binder.tensorOperand(operand) ||
-                      binder.tensorResultType(resultType)) {
-                    return failure();
-                  }
+  patterns.onOp(
+      "Not", 1, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        Value operand;
+        if (binder.tensorOperand(operand) ||
+            binder.tensorResultType(resultType)) {
+          return failure();
+        }
 
-                  auto loc = binder.getLoc();
-                  auto operandTy =
-                      cast<Torch::ValueTensorType>(operand.getType());
-                  auto eTy = operandTy.getDtype();
-
-                  if (!eTy.isInteger(1)) {
-                    auto i1ty = rewriter.getI1Type();
-                    auto ty = rewriter.getType<Torch::ValueTensorType>(
-                        operandTy.getSizes(), i1ty);
-                    auto torchqTy = Torch::getScalarTypeForType(i1ty);
-                    Value tyConst = rewriter.create<Torch::ConstantIntOp>(
-                        binder.getLoc(), rewriter.getType<Torch::IntType>(),
-                        rewriter.getIntegerAttr(
-                            rewriter.getIntegerType(64),
-                            static_cast<int64_t>(torchqTy)));
-                    Value none = rewriter.create<Torch::ConstantNoneOp>(loc);
-                    Value cstFalse =
-                        rewriter.create<Torch::ConstantBoolOp>(loc, false);
-                    operand = rewriter.create<Torch::AtenToDtypeOp>(
-                        loc, ty, operand, tyConst,
-                        /*non_blocking=*/cstFalse, /*copy=*/cstFalse,
-                        /*memory_format=*/none);
-                  }
-                  rewriter.replaceOpWithNewOp<Torch::AtenBitwiseNotOp>(
-                      binder.op, resultType, operand);
-                  return success();
-                });
+        auto loc = binder.getLoc();
+        auto operandTy = cast<Torch::ValueTensorType>(operand.getType());
+        auto eTy = operandTy.getDtype();
+
+        if (!eTy.isInteger(1)) {
+          auto i1ty = rewriter.getI1Type();
+          auto ty = rewriter.getType<Torch::ValueTensorType>(
+              operandTy.getSizes(), i1ty);
+          auto torchqTy = Torch::getScalarTypeForType(i1ty);
+          Value tyConst = rewriter.create<Torch::ConstantIntOp>(
+              binder.getLoc(), rewriter.getType<Torch::IntType>(),
+              rewriter.getIntegerAttr(rewriter.getIntegerType(64),
+                                      static_cast<int64_t>(torchqTy)));
+          Value none = rewriter.create<Torch::ConstantNoneOp>(loc);
+          Value cstFalse = rewriter.create<Torch::ConstantBoolOp>(loc, false);
```

Comment:
If all of this is done by clang-format, then it's strange that it was not caught by CI before.

Reply:
It seems that Lint Check has not been run by CI?